.prospector.yml

```diff
@@ -0,0 +1,82 @@
+output-format: grouped
+
+# strictness: veryhigh
```

Comment:
Not enabled the strictness of the `prospector` right now. We will enable and use it as required.